cs/client.py

```diff
@@ -431,4 +432,14 @@ def read_config(ini_group=None):
     if missings:
         raise ValueError("the configuration is missing the following keys: "
                          ", ".join(missings))
+
+    # convert booleans values.
+    if isinstance(config['verify'], string_type):
+        try:
+            verify = strtobool(config['verify'])
+        except ValueError:
+            pass
```

Comment:
Should we not throw an error message if the verify configuration is not correctly set ?

Reply:
nope. docs.python-requests.org/en/master/user/advanced/?#ssl-cert-verification

Reply:
`verify` is of type `Union[str, bool]`